--- docker-entrypoint.sh	2015-02-02 09:59:28.145834092 +0100
+++ new_docker-entrypoint.sh	2015-02-02 10:00:12.169832251 +0100
@@ -8,6 +8,9 @@
 		gosu postgres initdb
 		
 		sed -ri "s/^#(listen_addresses\s*=\s*)\S+/\1'*'/" "$PGDATA"/postgresql.conf
+
+                # Set max_prepared_transactions to 100 cf http://documentation.bonitasoft.com/database-configuration-1#postgresqlspec
+                sed -i 's/^.*max_prepared_transactions\s*=\s*\(.*\)$/max_prepared_transactions = 100/' "$PGDATA"/postgresql.conf
 		
 		# check password first so we can ouptut the warning before postgres
 		# messes it up
